Ubuntu1804-readme.md

```diff
@@ -1 +1,297 @@
+# Ubuntu 18.04.5 LTS
+- Image Version: 20200820.1
 
+## Installed Software
+### Basic packages
+- dnsutils
+- iproute2
+- iputils-ping
+- libunwind8
+- locales
+- openssh-client
+- tzdata
+- upx
+- zstd
+- libxkbfile-dev
+- pkg-config
+- libsecret-1-dev
+- libxss1
+- libgconf-2-4
+- dbus
+- xvfb
+- libgbm-dev
+- libgtk-3-0
+- tk
+- fakeroot
+- dpkg
+- rpm
+- xz-utils
+- xorriso
+- zsync
+- gnupg2
+- lib32z1
+- texinfo
+- libsqlite3-dev
+- libc++-dev
+- libc++abi-dev
+- curl
+- file
+- ftp
+- jq
+- netcat
+- ssh
+- parallel
+- rsync
+- shellcheck
+- sudo
+- telnet
+- time
+- unzip
+- zip
+- wget
+- m4
+- bison
+- flex
+- patchelf
+- bzip2
+- sqlite3
+- brotli
+- yamllint
+- libcurl3
+
+### Language and Runtime
+- GNU C++ 7.5.0
+- GNU C++ 8.4.0
+- GNU C++ 9.3.0
+- GNU Fortran 8.4.0
+- GNU Fortran 9.3.0
```

Comment:
maybe specify versions with comma? Like we do for .Net versions or for NVM cached versions on macOS
![image](https://user-images.githubusercontent.com/8777209/92091552-94e5dd80-edd9-11ea-89de-e958322474aa.png)

Reply:
Done